server/src/main/java/io/crate/execution/engine/collect/sources/TableFunctionCollectSource.java

```diff
@@ -87,7 +87,7 @@ public TableFunctionCollectSource(Functions functions) {
             topLevelInputs.add(ctx.add(symbol));
         }
 
-        Iterable<Row> result = functionImplementation.evaluate(txnCtx, inputs.toArray(new Input[0]));
+        Iterable<Row> result = functionImplementation.evaluate(txnCtx, null, inputs.toArray(new Input[0]));
```

Comment:
The `nodeCtx` should be used here instead of passing `null`

Reply:
well spotted! thank you very much.